### ISSUE TITLE
Invalid trait conflict errors.

### ISSUE THREAD

**Issue by Fischer-Bjoern** [pdepend/pdepend#367]
In relation to #190 

```php
trait A {
    function Test() {
    }
}
```

```php
trait B {
    use A;
}
```

```php
class C {
    use A;
    use B {
        A::Test insteadof B;
    }
}
```

Causes an error:

> Trait method Test has not been applied, because there are collisions with other trait methods on \global\C.

Note that with PHP 7.3 in this case the 'insteadof' will not even be needed any more.

This happens with PDepend 2.5.2.

**Comment by kylekatarnls** [comment]
#458 should fix this bug, please try to require `"pdepend/pdepend": "dev-feature/issue-367-fix-trait-instead"` and check if it solves your issue.

**Comment by Fischer-Bjoern** [comment]
Yes, pdepend works correctly, when I use the mentioned branch.

**Comment by kylekatarnls** [comment]
Great :) this will be released in PDepend 2.7.2.
